src/net_processing.cpp

```diff
@@ -1926,6 +1926,20 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
         return true;
     }
 
+    // VeriBlock: if POP is not enabled, ignore POP-related P2P calls
+    if (VeriBlock::isPopActive()) {
+        LOCK(cs_main);
+        auto& state = *State(pfrom->GetId());
+        // ignore all POP-related messages until sync is completed
+        if(!ChainstateActive().IsInitialBlockDownload() 
+            && (state.pindexBestKnownBlock == nullptr || ChainActive().Height() > state.pindexBestKnownBlock->nHeight - 20)) {
```

Comment:
also do similar check with `pindexBestKnownHeader` - it's a best known header in our local peer